### ISSUE TITLE
Configuration check of CStools

### ISSUE THREAD

**Issue by Eocin1** [thoughtspot/cs_tools#63]
### First Stop

- [X] I have read and agreed to the ThoughtSpot Dev Tools EULA.
- [X] I understand that if I have a critical issue with ThoughtSpot, I should reach out to my designated Account Executive and Solutions Consultant team instead of cs_tools.
- [X] I searched the cs_tools documentation, with the integrated search.

### Platform Configuration

 Info snapshot taken on 2023-05-31                                                                          │
│                                                                                                                      │
│               CS Tools: 1.4.6                                                                                        │
│         Python Version: Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]    │
│            System Info: Windows (detail: Windows-10-10.0.22621-SP0)                                                  │
│      Configs Directory: C:\Users\ {anonymous} \AppData\Roaming\cs_tools                                              │
│    Activate VirtualEnv: C:\Users\ {anonymous} \AppData\Roaming\cs_tools\.cs_tools\Scripts\Activate.ps1               │
│          Platform Tags: win-amd64

### Description

Cs_tools config check --config CSTool_dev
provides unboundLocalError 
log collected and attached.
[2023-05-31T14_33_37.log](https://github.com/thoughtspot/cs_tools/files/11614941/2023-05-31T14_33_37.log)

**Comment by boonhapus** [comment]
@Eocin1 It looks like you're getting a 404 (page not found) for `session/info` , are you setting up a new configuration or did this start on an existing configuration?

**Comment by Eocin1** [comment]
I am building on the existing configuration .

**Comment by boonhapus** [comment]
@Eocin1 Updating to 1.4.7 should fix the issue with `UnboundLocalError` which can likely give you more insight into the HTTP 404. The only other thing I can think of for 404 is if your ThoughtSpot version is <8.4.1 ... we raised this requirement due to migrating off all of the internal APIs.

ThoughSpot Software version 8.4.1 was the first TSE release, which showed an incredible effort in standardizing the API layer. This version was released in November 2022.
